src/napi/node_api_value.c

```diff
@@ -107,8 +106,11 @@ napi_status napi_create_object(napi_env env, napi_value* result) {
      * error flag.                                                            \
      */                                                                       \
     jerryx_create_handle(jval_error);                                         \
-                                                                              \
-    iotjs_jval_set_property_jval(jval_error, "code", jval_code);              \
+    /** code has to be an JS string type, thus it can not be an number 0 */   \
+    if (code != NULL) {                                                       \
```

Comment:
Validate it in JerryScript API?

Reply:
it has been validated on next line only if code is not NULL - that is code is given a value.